default.cfg.example

```diff
@@ -84,8 +84,27 @@ hideCoins = True
 #If maxtolend is 0: check for maxpercenttolend.
 #If maxpercenttolend is 0: 100% is going to be lent.
 #If maxtolendrate is set to more than 0: the maxtolend or maxpercenttolend will be used when then rate is less or equal to the maxtolendrate. if set to 0 the bot will use the maxtolend or maxpercenttolend all the time.
+# Please configure via the coinconfig line below OR individual coins in the [BTC], [CLAM], etc section. NOT both.
+# coinconfig takes precedence
 #coinconfig = ["BTC:0.18:1:0:0:0","CLAM:0.6:1:0:0:0"]
 
+#[BTC]
+#minloansize = 0.01
+#mindailyrate = 0.18
+#maxactiveamount = 1
+#maxtolend = 0
+#maxpercenttolend = 0
+#maxtolendrate = 0
+#
+#[CLAM]
```

Comment:
New sections such as [CLAM] or [BTC] need to go last and cannot split existing sections, such as [API] or [BOT], since that would make the json and webserver configs below part of [CLAM]